src/js/common/schemaform/FormPage.jsx

```diff
@@ -112,13 +127,28 @@ class FormPage extends React.Component {
 
   render() {
     const { route, params, form } = this.props;
+    const { profile } = this.props.user;
     let {
       schema,
       uiSchema
     } = form.pages[route.pageConfig.pageKey];
 
     let data = form.data;
-
+    let savedAt;
```

Comment:
Might be cleaner to make a SaveStatus component and put this logic in it.

Reply:
Good point, change made.